packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -33,6 +34,7 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 		reusableBlocks,
 		hasUploadPermissions,
 		canUseUnfilteredHTML,
+		userCanCreatePages,
```

Comment:
Nit: I like `canUserCreatePages` better 😅, it's more like a boolean to me.

Reply:
I agree other than the fact that the Nav block already uses the `userCanCreatePages` pattern. Happy to raise a follow up to update this though as I agree your suggestion might be preferable.